src/com/maddyhome/idea/vim/group/ChangeGroup.java

```diff
@@ -421,6 +422,8 @@ private void initInsert(@NotNull Editor editor, @NotNull DataContext context, @N
 
       VisualGroupKt.updateCaretState(editor);
     }
+
+//    VimHighlightedYank.Companion.clearAllYankHighlighters();
```

Comment:
You might have not noticed this before but I guess this it the same case as with yank ` I think this would be a bad code to inject extension-specific calls into the core code`. Should I do the same thing as I did with yank, this is listener pattern?
There is a note in original plugin that when user enters the insert mode all highlights should be removed, so originally put the line here